front/tests/utils/api-setup.ts

```diff
@@ -260,3 +264,46 @@ export async function setStdcmEnvironment(stdcmEnvironment: StdcmSearchEnvironme
     'Failed to update STDCM configuration environment'
   );
 }
+
+/**
+ * Retrieve a towed rolling stock by name.
+ *
+ * @param towedRollingStockName - The name of the towed rolling stock to retrieve.
+ * @returns {Promise<TowedRollingStock >} - The matching towed rolling stock data .
+ */
+export const getTowedRollingStock = async (
```

Comment:
This would be more telling (and matches the syntax from editoast endpoints)
```suggestion
export const getTowedRollingStockByName = async (
```